starkinfra/issuingtokenrequest/__issuingtokenrequest.py

```diff
@@ -0,0 +1,40 @@
+from starkcore.utils.subresource import SubResource
+from starkinfra.utils import rest
+
+
+class IssuingTokenRequest(SubResource):
+    """# IssuingTokenRequest object
+        The IssuingTokenRequest object displays the information of the Token Requests created for your card tokenization.
```

Comment:
`The IssuingTokenRequest object displays the necessary information to proceed with the card tokenization.`